lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -112,11 +112,30 @@ def update_to_cocoapods_0_34
         # @todo   This can be removed for CocoaPods 1.0
         #
         def update_to_cocoapods_0_37_1
-          (native_targets - native_targets_to_embed_in).any? do |native_target|
+          targets_to_embed = native_targets.select do |target|
```

Comment:
@mrackwitz note that this fixes a bug with the migration, since `native_targets_to_embed_in` can be empty if targets are already integrated